docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.7'
+
+services:
+  nutstash:
+    image: nutstash
+    build:
+      context: .
+      dockerfile: Dockerfile
+      platforms:
+        - "linux/amd64"
+        - "linux/arm64"
```

Comment:
In order to bring the app to the umbrel app store, the image needs to be available at lest for the two platforms `linux/amd64` and `linux/arm64`.